appinventor/components-ios/src/YailRuntimeError.swift

```diff
@@ -5,12 +5,31 @@
 
 import Foundation
 
+//@objc open class YailRuntimeError : NSException, Error, @unchecked Sendable {
+//  @objc public init(_ message: String, _ errorType: String) {
+//    super.init(name: errorType as NSString as NSExceptionName, reason: message, userInfo: nil)
+//  }
+//
+//  public required init?(coder aDecoder: NSCoder) {
+//    fatalError("init(coder:) has not been implemented")
+//  }
+//}
 @objc open class YailRuntimeError : NSException, Error, @unchecked Sendable {
   @objc public init(_ message: String, _ errorType: String) {
     super.init(name: errorType as NSString as NSExceptionName, reason: message, userInfo: nil)
   }
   
   public required init?(coder aDecoder: NSCoder) {
-    fatalError("init(coder:) has not been implemented")
+    let alert = UIAlertController(title: "Error", message: "YailRuntimeError occurred. \(aDecoder)", preferredStyle: .alert)
```

Comment:
Actually, in the case of YailRuntimeError, we can implement the NSCoder version. Since YailRuntimeError doesn't have any additional state, we only have to call through to the super implementation and pass the coder to it.